build-coordinator/src/main/java/org/jboss/pnc/core/builder/coordinator/bpm/BpmBuildScheduler.java

```diff
@@ -58,13 +57,18 @@
 
     private BpmCompleteListener bpmCompleteListener;
 
-    private final URL instanceUrl;
```

Comment:
It required valid URL also if you don't use BPM and it caused an error due to our env variable reference. Validation is still done in later stage, when you are connecting to BPM server. I prefer the first option but a compromise is required.